ports/nrf/examples/nrf9160_gps.py

```diff
@@ -0,0 +1,46 @@
+# GNSS sample for pca10090
+
+import network
+import socket
+
+n = network.NRF91()
```

Comment:
maybe this should be `network.Cellular()` so it can be consistent with other ports that may eventually provide such a network class?

Reply:
Great suggestion! I never came up with a good name for it.